ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/string/RPadTest.java

```diff
@@ -18,69 +18,134 @@
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+import io.confluent.ksql.util.BytesUtils;
+import java.nio.ByteBuffer;
 import org.junit.Test;
 
 public class RPadTest {
   private final RPad udf = new RPad();
 
   @Test
-  public void shouldPadInput() {
+  public void shouldPadInputString() {
     final String result = udf.rpad("foo", 7, "Bar");
     assertThat(result, is("fooBarB"));
   }
 
   @Test
-  public void shouldAppendPartialPadding() {
+  public void shouldPadInputBytes() {
+    final ByteBuffer result = udf.rpad(BYTES_123, 7, BYTES_45);
+    assertThat(BytesUtils.getByteArray(result), is(new byte[]{1,2,3,4,5,4,5}));
```

Comment:
The new unit tests for bytes don't seem to cover the case where target length is greater than start length and the difference is not an integer multiple of the length of the padding array.

Reply:
Doesn't `shouldAppendPartialPaddingBytes()` do this?

Reply:
Sorry my previous comment was incomplete -- the missing test coverage is the case where the padding needs to wrap around (pad multiple occurrences) but not an integer number of times. It's a minor point, though. Let's merge now to get this in!